sdk/communication/Azure.Communication.Common/src/MicrosoftTeamsUserIdentifier.cs

```diff
@@ -8,28 +8,38 @@ namespace Azure.Communication
     /// <summary>Represents a Microsoft Teams user.</summary>
     public class MicrosoftTeamsUserIdentifier : CommunicationIdentifier
     {
+        /// <summary>The id of the Microsoft Teams User identifier.</summary>
```

Comment:
Let's use the word `full` to distinguish Id better from UserId

```suggestion
        /// <summary>The full id of the Microsoft Teams User identifier.</summary>
```